src/main/java/org/snomed/aag/data/services/ProjectAcceptanceCriteriaService.java

```diff
@@ -266,9 +264,15 @@ public void delete(ProjectAcceptanceCriteria projectAcceptanceCriteria) {
 		if (criteria == null) {
 			return Collections.emptySet();
 		}
-		Set<CriteriaItem> criteriaItems = criteriaItemService.findAllByIdentifiers(criteria.getAllCriteriaIdentifiers());
-		criteriaItemSignOffService.markSignedOffItems(branchPath, criteria.getProjectIteration(), criteriaItems);
-		return criteriaItems;
+
+        Set<String> criteriaIdentifiers = criteria.getAllCriteriaIdentifiers();
+        if (criteriaIdentifiers.isEmpty()) {
+            return Collections.emptySet();
+        }
+
```

Comment:
I think there was a failing unit test or something. I added in an isEmpty check which I think is sensible behaviour.

Reply:
Yes, I expect the find method on the next line would fail without this 👍